OrbitVulkanLayer/VulkanLayerProducer.h

```diff
@@ -28,11 +29,22 @@ class VulkanLayerProducer {
   [[nodiscard]] virtual bool IsCapturing() = 0;
 
   // Use this method to enqueue a CaptureEvent to be sent to OrbitService.
-  virtual void EnqueueCaptureEvent(orbit_grpc_protos::CaptureEvent&& capture_event) = 0;
+  virtual bool EnqueueCaptureEvent(orbit_grpc_protos::CaptureEvent&& capture_event) = 0;
```

Comment:
And here. Yes, this is on purpose. In my view comments to justify a missing `[[nodiscard]]` are a bit overkill (probably because I don't like `[[nodiscard]]` to begin with...) if it's clear what the return value is (which it wasn't as the comment should be expanded as per the previous review comment), but I guess an extra explanation doesn't do any harm either.

Reply:
I agree with Davide. Having the comment about the return value should be fine.

Reply:
Yeah, I wouldn't explicitly explain why [[nodiscard]] is omitted here, but one can subtly hint at that in the comment for the return value: "Callers can check the return value if the event was actually enqueued, in case they need this information." Or something like that.

Reply:
Added that one.

Reply:
We are really in the realm of minor things and of preferences, but to me that's quite redundant ("can" + "in case they need") and also gets rid of "as the capture is in progress". If we really want to hint at it I would prefer something like
"Callers can use the return value to check if the event was actually enqueued as the capture is in progress".

Reply:
Done.